pallets/amm/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[dependencies]
+frame-support = { branch = "polkadot-v0.9.8", default-features = false, git = "https://github.com/paritytech/substrate" }
```

Comment:
let's always use single quote in `Cargo.toml` to make consistency across the project

Reply:
Replaced